src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -82,7 +82,13 @@ export class PredictiveTextStudioWorkerImpl
     name: string,
     contents: File
   ): Promise<number> {
-    const wordlist = await readExcel(await contents.arrayBuffer());
+    let wordlist: WordList = [];
+    if (/\.(tsv)$/i.test(name)) {
+      wordlist = await readTSV(contents);
+    } else {
```

Comment:
Should we add logic here to throw an error if the file is neither a .tsv file nor an Excel file? Not entirely sure what Excel files are supported currently